tests/pytests/functional/modules/test_chocolatey.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+from salt.exceptions import CommandExecutionError
+
+pytestmark = [
+    pytest.mark.destructive,
+    pytest.mark.skip_unless_on_windows,
+    pytest.mark.slow_test,
+    pytest.mark.windows_whitelisted,
+]
+
+
+@pytest.fixture(scope="module")
+def chocolatey(modules):
+    return modules.chocolatey
+
+
+@pytest.fixture()
+def clean(chocolatey):
+    chocolatey.unbootstrap()
+    try:
+        chocolatey_version = chocolatey.chocolatey_version(refresh=True)
+    except CommandExecutionError:
+        chocolatey_version = None
+    assert chocolatey_version is None
+    yield
+    chocolatey.unbootstrap()
```

Comment:
```suggestion
def clean(chocolatey):
    chocolatey.unbootstrap()
    try:
        chocolatey_version = chocolatey.chocolatey_version(refresh=True)
    except CommandExecutionError:
        chocolatey_version = None
    assert chocolatey_version is None
    try:
        yield
    finally:
        chocolatey.unbootstrap()
```
We always want `chocolatey.unbootstrap()` to run, even if the test fails, correct?

Reply:
Done